runtimes/runtimes/util/loggingUtil.ts

```diff
@@ -0,0 +1,68 @@
+import { Logging } from '../../server-interface'
+import { Connection } from 'vscode-languageserver/node'
+
+export type LogLevel = 'error' | 'warn' | 'info' | 'log' | 'debug'
+
+const logLevelMap: Record<LogLevel, number> = {
```

Comment:
Let's align log level enum with standard LSP `MessageType` interface 
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#messageType. We can use it here directly